### PR TITLE
PoC mobile worker login as webuser

### PR DESCRIPTION
**Pull request by benrudolph**
https://github.com/dimagi/commcare-hq/pull/9428

### CODE REVIEW

File: touchforms/formplayer/views.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import tempfile
 import os
 from . import api
-from touchforms.formplayer.api import DjangoAuth, get_raw_instance
+from touchforms.formplayer.api import DigestAuth, DjangoAuth, get_raw_instance
```
Comment (dimagimon) [inline touchforms/formplayer/views.py]: F401 'get_raw_instance' imported but unused

File: touchforms/formplayer/static/formplayer/script/xforgasm.js [hunk 2]
```diff
@@ -162,28 +157,25 @@ function WebFormSession(params) {
...
     }
 
-    self.serverRequest = function (params, callback, blocking) {
+    self.serverRequest = function (requestParams, callback, blocking) {
         var that = this;
         var url = that.urls.xform;
-        if (params.action === 'submit-all' && self.NUM_PENDING_REQUESTS) {
-            self.taskQueue.addTask(params.action, self.serverRequest, arguments, self)
+        if (requestParams.action === 'submit-all' && self.NUM_PENDING_REQUESTS) {
+            self.taskQueue.addTask(requestParams.action, self.serverRequest, arguments, self)
```
Comment (dimagimon) [inline touchforms/formplayer/static/formplayer/script/xforgasm.js]: Missing semicolon.

**Comment by czue** [comment]
this is cool and no blockers on my side - leaving a comment in the other PR

**Comment by czue** [comment]
(g2m when you fix lint errors)

**Pull request closed without merging**